slice/internal/util/testing/wrappers.go

```diff
@@ -170,11 +170,14 @@ func (w *WorkloadWrapper) ControllerReference(gvk schema.GroupVersionKind, name,
 
 type PodSetWrapper struct{ kueue.PodSet }
 
-func MakePodSet(name kueue.PodSetReference, count int) *PodSetWrapper {
+func MakePodSet(name kueue.PodSetReference, count int, replicas int) *PodSetWrapper {
```

Comment:
nit, you may consider making it ptr so that later when we start to support Job there is fewer changes. As Job will not set it.